### PR TITLE
Use a shared library for eslint

### PR DESCRIPTION
**Pull request by marcoacierno**
Use a shared lib instead of having to add eslint dependencies to all JS projects

### CODE REVIEW

**Comment by marcoacierno** [comment]
eslint on the gateway is failing, I will fix in another PR

**Comment by marcoacierno** [comment]
Fixed all gateway errors in #2343

File: association-frontend/package.json [hunk 1]
```diff
@@ -35,21 +35,15 @@
     "@graphql-codegen/typescript-operations": "^1.17.14",
     "@graphql-codegen/typescript-react-apollo": "^2.2.1",
     "@graphql-codegen/typescript-urql": "^2.0.3",
+    "@python-italia/eslint-config-pycon": "^1.0.4",
```
Comment (patrick91) [inline association-frontend/package.json:38]: nice

Comment (patrick91) [reply association-frontend/package.json:38]: we could one for all our js apps instead of just pycon maybe? :D

**Review by patrick91 — APPROVED** [review]

File: association-frontend/package.json (hunk 1 above)
Comment (marcoacierno) [reply association-frontend/package.json:38]: I can name the library "@python-italia/eslint-config-pythonit"

Comment (marcoacierno) [reply association-frontend/package.json]: Created `@python-italia/eslint-config` as global eslint config for python italia projects

Now we just need

```json
{
  "extends": ["@python-italia"]
}
```

to use it

**Pull request merged**